forui/lib/src/widgets/select/select_item.dart

```diff
@@ -0,0 +1,291 @@
+          // No feasible solution that doesn't involve a lot of complexity exists.
+          if (children.firstOrNull case final first?)
+            SelectContentData<T>(
+              style: style,
+              first: content.first,
+              enabled: enabled,
+              ensureVisible: content.ensureVisible,
+              child: first,
+            ),
+          ...children.skip(1),
+        ],
+      ),
+    );
+  }
+
+  @override
+  void debugFillProperties(DiagnosticPropertiesBuilder properties) {
+    super.debugFillProperties(properties);
+    properties
+      ..add(DiagnosticsProperty('style', style))
+      ..add(FlagProperty('enabled', value: enabled, ifTrue: 'enabled', ifFalse: 'disabled'));
+  }
+}
+
+/// A [FSelectSection]'s style.
+class FSelectSectionStyle with Diagnosticable, _$FSelectSectionStyleFunctions {
+  /// The enabled title's text style.
+  @override
+  final TextStyle enabledTitleTextStyle;
+
+  /// The disabled title's text style.
+  @override
+  final TextStyle disabledTitleTextStyle;
+
+  /// The padding around the title. Defaults to `EdgeInsetsDirectional.only(start: 15, top: 7.5, bottom: 7.5, end: 10)`.
+  @override
+  final EdgeInsetsGeometry titlePadding;
+
+  /// The section's items' style.
+  @override
+  final FSelectItemStyle itemStyle;
+
+  /// Creates a [FSelectSectionStyle].
+  FSelectSectionStyle({
+    required this.enabledTitleTextStyle,
+    required this.disabledTitleTextStyle,
+    required this.itemStyle,
+    this.titlePadding = const EdgeInsetsDirectional.only(start: 15, top: 7.5, bottom: 7.5, end: 10),
+  });
+
+  /// Creates a [FSelectSectionStyle] that inherits from the given [FColorScheme], [FStyle], and [FTypography].
+  FSelectSectionStyle.inherit({
+    required FColorScheme colorScheme,
+    required FStyle style,
+    required FTypography typography,
+  }) : this(
+         enabledTitleTextStyle: typography.sm.copyWith(color: colorScheme.primary, fontWeight: FontWeight.w600),
+         disabledTitleTextStyle: typography.sm.copyWith(
+           color: colorScheme.disable(colorScheme.primary),
+           fontWeight: FontWeight.w600,
+         ),
+         itemStyle: FSelectItemStyle.inherit(colorScheme: colorScheme, style: style, typography: typography),
+       );
+}
+
+/// A selectable item in a [FSelect] that can optionally be nested in a [FSelectSection].
+class FSelectItem<T> extends StatefulWidget with FSelectItemMixin {
+  /// The style. Defaults to the [FSelectItemStyle] inherited from the parent [FSelectSection] or [FSelect].
+  final FSelectItemStyle? style;
+
+  /// The value.
+  final T value;
+
+  /// True if the item is enabled. Disabled items cannot be selected, and is skipped during traversal.
+  ///
+  /// Defaults to the value inherited from the parent [FSelectSection] or [FSelect].
+  final bool? enabled;
+
+  /// The title.
+  final Widget title;
```

Comment:
I think title is a bit inconsistent here, it should either be named `label` or `child`. I think if we're trying to move towards using `child`, then that should be preferred.